src/logic/schedule-exporter/schedule-export.logic.ts

```diff
@@ -147,6 +157,18 @@ export class ScheduleExportLogic {
     workSheet.addRows(rows);
     workSheet.getColumn(1).width = 20;
     workSheet.eachRow((row, index) => {
+      if (
+        (index > headerLen && index <= nurseLastIndex) ||
+        (index > nurseLastIndex + 1 && index <= babysitterLastIndex)
+      ) {
+        row.eachCell((cell, colNumber) => {
+          const cellValue = cell.value?.toString() || "";
+          cell.style = this.getShiftStyle(
+            ShiftCode[cellValue] || ShiftCode.W,
+            verboseDates[colNumber - 2]
```

Comment:
Co oznacza ta -2? Może wynieść to do stałej i nazwij

Reply:
Okej, poprawię to, dzięki za uwagę :)
To -2 jest potrzebnym marginesem żeby data w danych odpowiadała poprawnej kolumnie (na ile pamiętam), dokładniej i dzaczego -2 mogę powiedzieć jak będę miał możliwość odpalić kod